doc/lms.mysql

```diff
@@ -3926,4 +3926,8 @@ INSERT INTO netdevicemodels (name, alternative_name, netdeviceproducerid) VALUES
 ('XR7', 'XR7 MINI PCI PCBA', 2),
 ('XR9', 'MINI PCI 600MW 900MHZ', 2);
 
-INSERT INTO dbinfo (keytype, keyvalue) VALUES ('dbversion', '2020122200');
+# ADDING SOME CONSTRAINTS WHITCH COULD NOT BE ADDED IN CREATE TABLE
+ALTER TABLE vlans ADD CONSTRAINT vlans_netnodeid_ukey UNIQUE (vlanid, netnodeid);
+ALTER TABLE vlans ADD CONSTRAINT vlans_netnodeid_fkey FOREIGN KEY (netnodeid) REFERENCES netnodes(id) ON DELETE SET NULL ON UPDATE CASCADE;
```

Comment:
Co to za wynalazek? Zmieniamy sposób dodawania kluczy obcych i wartości w pliku z pełnym schematem MySQL? Jeśli to kolejność tabel robi problem to trzeba zmienić trochę ich kolejność. Z drugiej strony, jeśli do takich problemów dochodzi to może to świadczyć, że rozwiązanie robi się zawiłe i przesadnie skomplikowane.
Nie przestajesz mnie zaskakiwać :)

Reply:
Nie mogę przesunąć tabeli nad netnodes nad vlans. Co mogę zrobić innego?

Reply:
Zaraz spróbuję przenieść tabelę vlans - może tak da radę.

Reply:
Udało się. Myślałem że jestem w błędnym kole.